docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MapValueIDVLookup.cs

```diff
@@ -0,0 +1,83 @@
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class MapValueIdvLookup
+    {
+        // Type for the IDataVIew that will be serving as the map
+        private class LookupMap
+        {
+            public float Value { get; set; }
+            public string Category { get; set; }
+
+        }
+
+        private class DataPoint
+        {
+            public float Price { get; set; }
+        }
+
+        private class TransformedData : DataPoint
+        {
+            public string PriceCategory { get; set; }
+        }
+
+        /// This example demonstrates the use of MapValue by mapping floats to strings, looking up the mapping in an IDataView. 
+        /// This is useful to map types to a grouping. 
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            var rawData = new[] {
+                new DataPoint() { Price = 3.14f },
+                new DataPoint() { Price = 2000f },
+                new DataPoint() { Price = 1.19f },
+                new DataPoint() { Price = 2.17f },
+                new DataPoint() { Price = 33.784f },
+
+            };
+
+            // Convert to IDataView
+            var data = mlContext.Data.LoadFromEnumerable(rawData);
+
+            // Create the lookup map data IEnumerable.   
+            var lookupData = new[] {
+                new LookupMap { Value = 3.14f, Category = "Low" },
+                new LookupMap { Category = "Low" , Value = 1.19f },
+                new LookupMap { Category = "Low" , Value = 2.17f },
+                new LookupMap { Category = "Medium", Value = 33.784f},
+                new LookupMap { Category = "High", Value = 2000f}
+
+            };
+
+            // Convert to IDataView
+            var lookupIdvMap = mlContext.Data.LoadFromEnumerable(lookupData);
+
+            // Constructs the ValueMappingEstimator making the ML.net pipeline
+            var pipeline = mlContext.Transforms.Conversion.MapValue("PriceCategory", lookupIdvMap, lookupIdvMap.Schema["Value"], lookupIdvMap.Schema["Category"], "Price");
+
+            // Fits the ValueMappingEstimator and transforms the data converting the Price to PriceCategory.
+            IDataView transformedData = pipeline.Fit(data).Transform(data);
+
+            // Getting the resulting data as an IEnumerable.
+            IEnumerable<TransformedData> features = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, reuseRowObject: false);
+
+            Console.WriteLine($" Price   PriceCategory");
+            foreach (var featureRow in features)
+                Console.WriteLine($"{featureRow.Price}\t\t{featureRow.PriceCategory}");
+
+            // TransformedData obtained post-transformation.
```

Comment:
>// TransformedData obtained post-transformation. [](start = 12, length = 48)

Everywhere else this is "// Expected output:"

Reply:
I think this is fine too, it matches the name of the type. 

---
In reply to: [271420199](https://github.com/dotnet/machinelearning/pull/3167#discussion_r271420199) [](ancestors = 271420199)

Reply:
ok.

---
In reply to: [271466196](https://github.com/dotnet/machinelearning/pull/3167#discussion_r271466196) [](ancestors = 271466196,271420199)